tests/storage/vm_export/utils.py

```diff
@@ -34,3 +40,23 @@ def get_manifest_url(vmexport_external_links, manifest_type):
     ).get("url")
     assert url, f"Manifest url '{manifest_type}' in vmexport external links {vmexport_external_links} not found"
     return url
+
+
+@contextmanager
+def create_blank_dv_by_specific_user(
```

Comment:
we should reduce adding wrapper functions for other functions.
this can be implemented in a fixture where client is controlled by request.param. e.g use_admin_client=True/False

Reply:
Having two separate fixtures 
`blank_dv_created_by_unprivileged_user` and 
`blank_dv_created_by_admin_user` 
makes the code more readable and explicit, so I prefer to have it this way.